server/src/game/role/mod.rs

```diff
@@ -309,13 +305,9 @@ impl Role{
     }
     pub fn has_suspicious_aura(&self, _game: &Game)->bool{
         match self {
-            Role::Politician => true,
             _ => false,
         }
     }
```

Comment:
Unnecessary match. Is your linter not yelling at you for this?

Reply:
what about when i add the four billion future roles that have a suspicious aura by default

Reply:
Bro never heard of `#[allow(blablabla)]`